### PR TITLE
Add code of conduct

### PR DESCRIPTION
**Pull request by ajzawawi**

### CODE REVIEW

**Comment by itaisteinherz** [comment]
Before I can merge this pull request, it would be great if you could change #18 according to my requests.

**Comment by ajzawawi** [comment]
@itaisteinherz I updated the code of conduct.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,45 +1,42 @@
...
+
+By adopting this Code of Conduct, project maintainers commit themselves to fairly and consistently
+applying these principles to every aspect of managing this project. Project maintainers who do not
+follow or enforce the Code of Conduct may be permanently removed from the project team.
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual
+is representing the project or its community.
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting
+a project maintainer at [INSERT EMAIL ADDRESS]. All complaints will be reviewed and investigated
```
Comment (itaisteinherz) [inline CODE_OF_CONDUCT.md]: Please change [INSERT EMAIL ADDRESS] to my email address (itaisteinherz@gmail.com).

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -1,45 +1,42 @@
...
+and will result in a response that is deemed necessary and appropriate to the circumstances.
+Maintainers are obligated to maintain confidentiality with regard to the reporter of an incident.
+
+This Code of Conduct is adapted from the Contributor Covenant, version 1.3.0,
+available from http://contributor-covenant.org/version/1/3/0/
```
Comment (itaisteinherz) [inline CODE_OF_CONDUCT.md]: It would be great you could change this link to use the markdown syntax.

**Comment by ajzawawi** [comment]
@itaisteinherz Updated both.

**Pull request merged**